src/common/registry/test/fetch/key.fetch.spec.ts

```diff
@@ -76,7 +76,33 @@ describe('Keys', () => {
     const result = await fetchService.fetch(address, expected.operatorIndex);
 
     expect(result).toEqual([expected]);
-    expect(mockCall).toBeCalledTimes(2);
+    expect(mockCall).toBeCalledTimes(3);
+  });
+
+  test('fetch all operator keys with reorg', async () => {
+    const expected = { operatorIndex: 1, index: 0, moduleAddress: address, ...key };
+
+    mockCall
```

Comment:
i think we dont use key.fetch service now 
we use in kapi now key-batch.fetch service.

Reply:
i mean maybe we need tests for  key-batch.fetch instead

Reply:
I made a fresh commit and implemented module level tests there